cdm/src/test/java/ucar/nc2/dataset/conv/TestSimpleGeom.java

```diff
@@ -0,0 +1,91 @@
+package ucar.nc2.dataset.conv;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ucar.ma2.Array;
+import ucar.nc2.Variable;
+import ucar.nc2.constants.AxisType;
+import ucar.nc2.constants.CF;
+import ucar.nc2.constants._Coordinate;
+import ucar.nc2.dataset.CoordinateAxis;
+import ucar.nc2.dataset.CoordinateAxisTimeHelper;
+import ucar.nc2.dataset.NetcdfDataset;
+import ucar.nc2.time.Calendar;
+import ucar.nc2.time.CalendarDate;
+import ucar.unidata.util.test.TestDir;
+
+import java.io.IOException;
+import java.lang.invoke.MethodHandles;
+import java.util.List;
+
+import static java.lang.String.format;
+
+public class TestSimpleGeom {
+	
+    private static final Logger logger = LoggerFactory.getLogger(MethodHandles.lookup().lookupClass());
+    private final String cfConvention = CF1Convention.class.getName();
+    
+    @Test
+    public void testLine() throws IOException {
+    	
+    	String failMessage, found, expected;
+        boolean testCond;
+
+        String tstFile = TestDir.cdmLocalTestDataDir + "dataset/SimpleGeos/hru_soil_moist_vlen_3hru_5timestep.nc";
+
+        // open the test file
+        NetcdfDataset ncd = NetcdfDataset.openDataset(tstFile);
+
+        // make sure this dataset used the cfConvention
+        expected = cfConvention;
+        found = ncd.getConventionUsed();
+        testCond = found.equals(expected);
+        failMessage = format("This dataset used the %s convention, but should have used the %s convention.", found, expected);
+        Assert.assertTrue(failMessage, testCond);

```

Comment:
It's pretty common to statically import the assertions (assertTrue, assertEquals, etc) so that you can reference them without using the `Assert.` prefix. Unless there's a convention prohibiting that in the thredds repo, I'd say you should feel free to statically import them in the future. Just an option, not an obligation.